### PR TITLE
Issue #8 - Update Readme

### PR DESCRIPTION
**Pull request by metalcanine**
Initial outline for readme, will update this branch and pr as we close out some of the other issues, like figuring out project structure, dependencies/imports, etc.

### CODE REVIEW

**Comment by metalcanine** [comment]
Most of this is lifted from the Gatsby project readme and the fxa readme and is largely guess work.

**Comment by metalcanine** [comment]
Added instructions for development workflow. Still need to cover:
- [ ] Deployment
- [ ] Documentation

File: README.md [hunk 1]
```diff
@@ -1,39 +1,46 @@
...
+[![<mozilla-services>](https://img.shields.io/circleci/build/gh/mozilla-services/updatebot?label=tests&style=flat-square)](https://circleci.com/gh/mozilla-services/updatebot)
+[![codecov](https://img.shields.io/codecov/c/gh/mozilla-services/updatebot?style=flat-square)](https://codecov.io/gh/mozilla-services/updatebot)
+
+Automated detection, patching, testing, and bug filing for updates in third-party libraries for Firefox.
+
+## 🐍 Initial Setup
+
+This project requires [Poetry](https://python-poetry.org/docs/) and a version of [Python](https://www.python.org/downloads/release/python-359/) at least greater than 3.5.
+
+You will also need to install [MySQL](https://www.mysql.com/downloads/).
```
Comment (tomrittervg) [inline README.md:14]: Perhaps "We talk to a database; currently MySQL is supported. Copy the local config file with `cp ...` and configure the database connection parameters. Updatebot will automatically create and populate the database with its structure and required data."

File: README.md [hunk 2]
```diff
@@ -1,39 +1,46 @@
...
+
+`poetry install`
+
+and then
+
+`poetry run automation.py`
+
+## 🤖 Testing
+
+Testing is handled in a single step of `poetry run test.py`
```
Comment (tomrittervg) [inline README.md:32]: I had to do `poetry run ./test.py`

File: README.md [hunk 3]
```diff
@@ -1,39 +1,46 @@
...
+
+## 👩‍🏭 Getting Started
+
+To get started developing updatebot, or to run it locally all you'll need is to:
+
+`poetry install`
+
+and then
+
+`poetry run automation.py`
```
Comment (tomrittervg) [inline README.md:28]: i had to do `poetry run ./automation.py`

File: README.md [hunk 4]
```diff
@@ -1,39 +1,46 @@
...
+
+## 🤖 Testing
+
+Testing is handled in a single step of `poetry run test.py`
+
+For formatting code automatically please use `poetry run autopep8 --in-place --recursive .`
```
Comment (tomrittervg) [inline README.md:34]: Actually `poetry run autopep8 --in-place --recursive --ignore E402 .`

**Comment by metalcanine** [comment]
Updated the readme based on your feedback, thanks Tom!

File: README.md [hunk 5]
```diff
@@ -1,39 +1,24 @@
...
+
+This project requires [Poetry](https://python-poetry.org/docs/) and a version of [Python](https://www.python.org/downloads/release/python-359/) at least greater than 3.5.
+
+We talk to a database; currently [MySQL](https://www.mysql.com/downloads/) is supported. Copy the local config file with `cp localconfig.py.example localconfig.py` and configure the database connection parameters. Updatebot will automatically create and populate the database with its structure and required data.
+
+To get started developing updatebot, or to run it locally you'll need to run `poetry install` and then `poetry run ./automation.py`
+
+Testing is handled in a single step of `poetry run ./test.py`
+
+For formatting code automatically please use `poetry run autopep8 --in-place --recursive --ignore E402 .`
```
Comment (tomrittervg) [inline README.md:18]: Do we also need to ignore 501?

File: README.md [hunk 6]
```diff
@@ -1,39 +1,24 @@
...
+
+For linting the codebase run `poetry run flake8 .`
```
Comment (tomrittervg) [inline README.md:20]: Don't we also need --ignore=E501,E402 like in the circleci config file?

**Pull request merged**